cmd/agentctl/commands/import.go

```diff
@@ -43,73 +43,97 @@ var (
 	timeout  uint
 )
 
-func init() {
-	RootCmd.AddCommand(importConfig)
-	importConfig.PersistentFlags().UintVar(&txops, "txops", 128,
+func NewImportCommand(cli *AgentCli) *cobra.Command {
+	opts := ImportOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "import [file]",
+		Aliases: []string{"i"},
+		Args:    cobra.ExactArgs(1),
+		Short:   "Import config data from file",
+		Example: `
+ Import file contents example:
+  $ cat input.txt
+  config/vpp/v2/interfaces/loop1 {"name":"loop1","type":"SOFTWARE_LOOPBACK"}
+  config/vpp/l2/v2/bridge-domain/bd1 {"name":"bd1"}
+
+ To import it into Etcd, run:
+  $ agentctl import input.txt
+
+ To import it via gRPC, use --grpc flag:
+  $ agentctl import --grpc=localhost:9111 input.txt
+
+FILE FORMAT
+
+ Contents of the import file must contain single key-value pair per line:
+
+    <key1> <value1>
+    <key2> <value2>
+    ...
+    <keyN> <valueN>
+
+ Empty lines and lines starting with '#' are ignored.
+
+KEY FORMAT
+
+ Keys can be defined in two ways:
+
+    - full: 	/vnf-agent/vpp1/config/vpp/v2/interfaces/iface1
+    - short:	config/vpp/v2/interfaces/iface1
```

Comment:
I could not make the `input` command work properly with `short` key, perhaps it needs more detailed help. Using it as it returns `no model registered for key` error. Below is written that it needs `--label` flag, but where? Using it with import returns `Error: unknown flag: --label`. The `full` key works with no issues.

Reply:
works with the latest commit, however, please update the flag in help to `--service-label `